src/plugins/wizard/public/application/app.scss

```diff
@@ -4,10 +4,10 @@
   padding: 0;
   display: grid;
   grid-template-rows: min-content 1fr;
-  grid-template-columns: 420px 1fr;
+  grid-template-columns: 470px 1fr;
```

Comment:
do we want to avoid making changes with pixels. could we use rem?

Reply:
This is the width of the side-panel. That unfortunately is not a good spot to use `rem` units which are more useful for fonts since we want the sizing to depend on the sizing of the parent element. This was changed to match the updated mocks.